website/src/components/Meta.js

```diff
@@ -1,43 +1,52 @@
 import GlobalsContext from 'contexts/GlobalsContext';
 import { META } from 'lib/constants';
-import { nextLoader } from 'lib/image-loaders';
 import { NextSeo } from 'next-seo';
 import Head from 'next/head';
 import { useContext } from 'react';
 
 export default function Meta({ title, description, image, seo }) {
   const globals = useContext(GlobalsContext);
 
+  // make sure image is absolute
+  function imagePath(imageUrl) {
+    // if relative, make absolute so that twitter doesn't complain
+    if (
+      imageUrl.indexOf('http://') === -1 &&
+      imageUrl.indexOf('https://') === -1
+    ) {
+      imageUrl = META.url + imageUrl;
```

Comment:
is it possible there's a situation where `imageUrl` is relative and without a leading slash and we need to add a slash between `META.url` and `imageUrl`? Also should we expect users/devs to always set the `META.url` without a trailing slash or do we need to check/sanitize that as well?

Reply:
good feedback, ty. agree we should check/document.